### PR TITLE
ZCS-816 Developer set-up instructions

### PR DESCRIPTION
**Pull request by gordyt**
Added basic instructions for setting up a machine for
doing ZCS development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,3 +129,161 @@ Then just run `./build.pl`.
...
+
+
+1. Create `/home/zimbra` and make `zimbra` the owner.
+
+		sudo mkdir /home/zimbra
+		sudo chown zimbra:zimbra /home/zimbra
+
+2. Install  `git`, `ant`, and `ant-contrib`.
+
+		sudo apt-get install git and ant-contrib
```
Comment (silentsakky) [inline README.md]: typo 
-and +ant

Comment (gordyt) [inline README.md]: Thank you! I noticed that and two other errors or omissions right after submitting the pull request.  Please refresh.

File: README.md [hunk 2]
```diff
@@ -129,3 +129,165 @@ Then just run `./build.pl`.
...
+
+	git clone git@github.com:Zimbra/zm-mailbox.git
+
+The following sub-directories `zm-mailbox` build and deploy separately:
+
+	client/
+	common/
+	milter-conf/
+	native/
+	soap
```
Comment (fatkudu) [inline README.md]: Good to be consistent - finish with '/' or without.

File: README.md [hunk 3]
```diff
@@ -129,3 +129,165 @@ Then just run `./build.pl`.
...
+* Back as the `zimbra` user, invoke the following command from `zm-mailbox/common`:
+
+		ant deploy publish-local
+
+* Then from the `zm-mailbox/store` directory:
+
+		ant deploy update-ldap-schema
+
+
+Your ZCS development server should not be running with the new attribute(s).  You can test that
```
Comment (fatkudu) [inline README.md]: s/not/now/

**Comment by gordyt** [comment]
Thank you Gren!  Have updated to address the issues that you noted.

File: README.md [hunk 4]
```diff
@@ -129,3 +129,165 @@ Then just run `./build.pl`.
...
+	zmprov ms `zmhostname` zimbraDNSMasterIP DNS-SERVER-IP-ADDRESS
+
+You may receive the following error when trying to send email:
+
+	No SMTP hosts available for domain
+
+If this occurs, you need to manually configure `zimbraSmtpHostname` for your domain(s).
+To configure `zimbraSmtpHostname`, do the following as the `zimbra` user:
+
+	zmprov md DOMAIN-NAMNE zimbraSmtpHostname `zmhostname`
```
Comment (plobbes) [inline README.md]: typo: DOMAIN-NAMNE to DOMAIN-NAME ?

Comment (gordyt) [inline README.md]: Thanks Phil! Updated PR with fix.

File: README.md [hunk 5]
```diff
@@ -129,3 +129,165 @@ Then just run `./build.pl`.
...
+
+* Then from the `zm-mailbox/store` directory:
+
+		ant deploy update-ldap-schema
+
+
+Your ZCS development server should now be running with the new attribute(s).  You can test that
+by querying them and modifying them with `zmprov`.  You can `git add ...` and `git commit`
+your changes now.
+
```
Comment (plobbes) [inline README.md]: nit: remove trailing whitespace / blank lines?

Comment (gordyt) [inline README.md]: Thanks Phil!  Fix has been pushed.

**Pull request merged**